IceCubesApp/App/Tabs/Settings/InstanceInfoView.swift

```diff
@@ -44,4 +46,10 @@ public struct InstanceInfoSection: View {
       .listRowBackground(theme.primaryBackgroundColor)
     }
   }
+  
+  private func format(_ int: Int) -> String {
+    let formatter = NumberFormatter()
+    formatter.numberStyle = .decimal
```

Comment:
It's supposed to follow the current user locale no?

Reply:
Don't know if the NumberFormatter should do this. BUt setting the language in the simulator to German shows the formatting I captured in the screenshot, and that's definitely not german.

Reply:
I thought that this was assumed by the formatter but apparently not. 

I'll add the `formatter.locale = .current` later. I have seen this work for Dutch locale in the simulator, which, like German, uses . as thousands separator.

Reply:
Ah, wait. It's not the language, it's the region setting. Did you change that to Germany, @sothawo ?

Reply:
And while we're here, would be good to have this formatter for the followers / following / posts count on user profile too now?

Reply:
Aaaaaaarghh. I changed the language not the region. Setting the region to Germany, it's alright. Sorry for the false alarm